lib/matplotlib/figure.py

```diff
@@ -2001,6 +2000,14 @@ def __init__(self, parent, subplotspec, *,
         if parent._layoutgrid is not None:
             self.init_layoutgrid()
 
+    @property
+    def dpi(self):
+        return self._parent.dpi
+
+    @dpi.setter
```

Comment:
not sure we want the setter here?  The getter makes sense as we may need this in the transforms, but do we want to be able to change the dpi of the whole figure from a sub figure?

On the other hard it seems realtively harmless and makes sure that subfigures feel as like figures as they can.

Reply:
I actually tried it, and it breaks some tests without the setter. I did not investigate any further though, so I don't know if that's a real problem or not.

Reply:
Happy to mess around with that.  I'd say dpi is one of our more fragile concepts.